run_tests.py

```diff
@@ -47,14 +50,15 @@ def run_all_tests(single_file, remote):
     utils.global_vars.init()
 
     if remote:
+        top_issue_num = json.loads(requests.get("https://api.github.com/repos/ISISComputingGroup/IBEX/issues?per_page=1").content)[0]["number"]
```

Comment:
Can we use the IBEX/issues constant here as well

Reply:
I don't think so as it has the slash afterwards which isn't in the URL, but i'll pull the entire URL out as a constant.